.travis.yml

```diff
@@ -1,4 +1,12 @@
 sudo: false
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - g++-4.8
```

Comment:
According to https://github.com/wadey/node-microtime/issues/34#issuecomment-138529797, yes. I'd be happy to prove me wrong though. I don't like it any more than you do!